src/avram/migrator/runner.cr

```diff
@@ -122,6 +122,14 @@ class Avram::Migrator::Runner
     end
   end
 
+  def rollback_to(last_version : Int64)
+    setup_migration_tracking_tables
+    subset = migrated_migrations.select do |mm|
+      mm.new.version.to_i64 > last_version
+    end
+    subset.reverse.each &.new.down
```

Comment:
What do you think about printing a message after this like "Done rolling back to #{last_ version}"?

Reply:
Fine by me